Author: Eric Cooper <ecc@cmu.edu>
Description:
 The -b option doesn't turn off the bar display as the man page says it should
 (Closes: #445991).

diff -Naurp iftop.orig/options.c iftop/options.c
--- iftop.orig/options.c	2008-09-07 23:53:08.000000000 +0000
+++ iftop/options.c	2009-01-24 22:58:21.000000000 +0000
@@ -302,7 +302,7 @@ void options_read_args(int argc, char **
                 break;
 
             case 'b':
-                config_set_string("show-bars", "true");
+                config_set_string("show-bars", "false");
                 break;
 
             case 'B':
